File: src/test/java/org/snake/models/basic/PointTest.java
package org.snake.models.basic;

import org.junit.Test;

import static org.junit.Assert.*;

public class PointTest {

    @Test
    public void testToString() {
        // Arrange
        Point point = new Point(1, 1);

        // Act & Assert
        assertEquals("Point{x=1, y=1}", point.toString());
    }
}

File: src/test/kotlin/org/snake/models/basic/PointTest.kt
package org.snake.models.basic

import org.junit.Assert
import org.junit.Test
import snake.models.basic.Point

class PointTest {
    @Test
    fun testToString() {
        // Arrange
        val point = Point(1, 1)

        // Act & Assert
        Assert.assertEquals("Point{x=1, y=1}", point.toString())
    }
}